modules/unsupported/mbstyle/src/main/java/org/geotools/mbstyle/BackgroundMBLayer.java

```diff
@@ -42,8 +43,7 @@
     private static String type = "background";
 
     public BackgroundMBLayer(JSONObject json) {
-        super(json);
-
+        super(json, new MBObjectParser(BackgroundMBLayer.class));
```

Comment:
MBObjectParser likes to report who is doing the parsing; so we have to pass it in.

The alternative is implementing a mixin with default methods; but it is hard to avoid leaking public methods.